src/components/AvatarGroup/AvatarGroupCounter.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import cx from "classnames";
+import Counter from "../Counter/Counter";
+import styles from "./AvatarGroupCounter.module.scss";
+
+const AvatarGroupCounter = ({
+  counterTooltipAvatars,
+  counterProps,
+  counterTooltipCustomProps,
+  avatarSize,
+  avatarType
+}) => {
+  const {
+    color: counterColor = Counter.colors.LIGHT,
+    count: counterValue = counterTooltipAvatars.length,
```

Comment:
its a bit confusing and I am not sure we need it.... count is a data in counter props I am not understand...

Reply:
count belongs to Counter props - it's the prop for setting counter value. We give ability to set custom counter value, Evgeniy asked for this. It's standard prop for such components, see `total` in MUI https://mui.com/material-ui/api/avatar-group/
About the name - I also don't like "count" too much, but that's how the prop is called in Counter, so don't want to confuse users with renaming it